test/integration/request-test.coffee

```diff
@@ -41,85 +41,105 @@ describe('Sending \'application/json\' request', ->
 )
 
 
-describe('Sending \'multipart/form-data\' request', ->
-  runtimeInfo = undefined
-  contentType = 'multipart/form-data'
-
-  beforeEach((done) ->
-    path = './test/fixtures/request/multipart-form-data.apib'
-
-    app = createServer({bodyParser: bodyParser.text({type: contentType})})
-    app.post('/data', (req, res) ->
-      res.json({test: 'OK'})
+[
+    name: 'API Blueprint'
+    path: './test/fixtures/request/multipart-form-data.apib'
+    supportsContentTypes: true,
+  ,
+    name: 'Swagger'
+    path: './test/fixtures/request/multipart-form-data.yaml'
+    supportsContentTypes: false,
+].forEach((apiDescription) ->
+  describe("Sending 'multipart/form-data' request described in #{apiDescription.name}", ->
+    runtimeInfo = undefined
+    contentType = 'multipart/form-data'
+
+    beforeEach((done) ->
+      app = createServer({bodyParser: bodyParser.text({type: contentType})})
+      app.post('/data', (req, res) ->
+        res.json({test: 'OK'})
+      )
+      dredd = new Dredd({options: {path: apiDescription.path}})
+
+      runDreddWithServer(dredd, app, (err, info) ->
+        runtimeInfo = info
+        done(err)
+      )
```

Comment:
Again, dummy server is created for each assertion. I think it is not necessary to create the same server over and over again (aka use `before`, not `beforeEach`) 🙂